packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -209,22 +209,27 @@ export class IndexedDbPersistence implements Persistence {
     private readonly clientId: ClientId,
     platform: Platform,
     private readonly queue: AsyncQueue,
-    serializer: JsonProtoSerializer
+    serializer: JsonProtoSerializer,
+    synchronizeTabs?: boolean
```

Comment:
I would make the argument non-optional in internal code. I believe this was optional in start() to not break inheritance.

Reply:
SGTM.